### ISSUE TITLE
WillPopScope' is deprecated and shouldn't be used. Use PopScope instead.

### ISSUE THREAD

**Issue by maheshmnj** [maheshmnj/navbar_router#48]
```
mahesh@Maheshs-MacBook-Air-M1-3 navbar_router % flutter analyze
Analyzing navbar_router...                                              

   info • 'WillPopScope' is deprecated and shouldn't be used. Use PopScope instead. This feature was deprecated after v3.12.0-1.0.pre •
          lib/src/navbar_router.dart:341:12 • deprecated_member_use
   info • 'WillPopScope.new' is deprecated and shouldn't be used. Use PopScope instead. This feature was deprecated after v3.12.0-1.0.pre •
          lib/src/navbar_router.dart:341:12 • deprecated_member_use
```
**Expected behavior**
No warnings

**Screenshots**

**Additional context**

https://docs.flutter.dev/release/breaking-changes/android-predictive-back

**Comment by bebaoboy** [comment]
Hello, are you currently on it? @maheshmnj

**Comment by maheshmnj** [comment]
Not started on that yet! If you are interested feel free to work on it, I would love to collaborate if required and review it.
Thanks

**Comment by maheshmnj** [comment]
Fixed and released in [v0.7.5](https://pub.dev/packages/navbar_router/changelog)
